argo-cd-bridges/gitlab/deploy/cronjob.yml

```diff
@@ -22,6 +22,13 @@ spec:
             resources: {}
             terminationMessagePath: /dev/termination-log
             terminationMessagePolicy: File
+            volumeMounts:
+              - name: plugin-volume
+                mountPath: /plugins
+          volumes:
+            - name: plugin-volume
+              configMap:
+                name: gitlab-argo-bridge-plugins
```

Comment:
Can you add an example to the README showing the format of this configMap?

Reply:
Yep - just added!